contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -650,7 +653,10 @@ library ValidationLogic {
       IERC20(reserve.variableDebtTokenAddress).totalSupply() == 0,
       Errors.VARIABLE_DEBT_SUPPLY_NOT_ZERO
     );
-    require(IERC20(reserve.aTokenAddress).totalSupply() == 0, Errors.ATOKEN_SUPPLY_NOT_ZERO);
+    require(
+      IERC20(reserve.aTokenAddress).totalSupply() == 0 && reserve.accruedToTreasury == 0,
```

Comment:
We could do a sum here (it saves 1 operation), but it's worsen readability

Reply:
I prefer it this way. A sum is strange, because they are different things